collections/_blogs/2020-09-05-announcing-the-meshery-adapter-for-open-service-mesh.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title:  "Announcing the Meshery Adapter for Open Service Mesh"
+date: 2020-09-04 10:17:05 +0000
+image: /assets/images/posts/2020-09-05-announcing-the-meshery-adapter-for-open-service-mesh/meshery-adapter-for-open-service-mesh.png
```

Comment:
OSM is pixelated this image. You can use the svg in this repo to avoid pixelation.

Try to remove some of the surrounding whitespace on the sides and top of the image, if you would. (I can do this tomorrow).